ruby/vampires.rb

```diff
@@ -0,0 +1,71 @@
+#Clint Thacker
+#CHI Nighhawks
+#Vampire Detection program
+
+puts "Number of employees to be processed:"
+count = gets.chomp.to_i
+
+until count == 0
+  puts "Name:"
+  name = gets.chomp
+  puts "Age:"
+  age = gets.chomp.to_i
+  puts "Birth year:"
+  birth_year = gets.chomp.to_i
+  true_age = 2017 - birth_year
+  puts "Enjoy some delicious garlic bread? (y/n)"
+  garlic_bread = gets.chomp
+  puts "Enroll in our mortal health insurance plan? (y/n)"
+  insurance = gets.chomp
+
+  puts "Please name any allergies (enter 'done' when finished):"
+  while user_input = gets.chomp
+    case user_input
+    when "done"
+      allergy_test_passed = true
+      break
+    when "sunshine"
+      allergy_test_passed = false
+      break
+    end
+  end
+  puts
+
+  if age == true_age
```

Comment:
Simplify this code by assigning the `age_test_passed` the comparison, this will set it to true or false automatically.

`age_test_passed = age == true_age`